x-pack/plugin/sql/qa/src/main/resources/agg.csv-spec

```diff
@@ -85,8 +85,8 @@ SELECT SUM(salary) FROM test_emp;
 aggregateWithCastPruned
 SELECT CAST(SUM(salary) AS INTEGER) FROM test_emp;
 
-  SUM(salary)
--------------
+ CAST(SUM(salary) AS INTEGER)
```

Comment:
Leftover from a different PR?

Reply:
It looks like it merged with the cast PR though I'm not sure why since it should be merged in master... I need to double check.